README.md

```diff
@@ -155,6 +155,29 @@ Alternatively, if you're looking for something more advanced that supports multi
 }
 ```
 
+### Zed
+
+To use Prettierd with Zed, you need to configure the `language_overrides` adding a `format_on_save` command for each of the languages you wish to be handling. Example configuration:
+```json
+{
+  "language_overrides": {
+    "TypeScript": {
+      "format_on_save": {
+        "external": {
+          "command": "/opt/homebrew/bin/prettierd",
```

Comment:
Can you add a note below or above the code block to make it clear that you're assuming `prettierd` was installed with homebrew and users may need to adjust the path if they're installing it in another way?